feature-libs/asm/core/store/selectors/customer-search.selectors.spec.ts

```diff
@@ -27,7 +27,7 @@ describe('Customer Search Results Selectors', () => {
     TestBed.configureTestingModule({
       imports: [
         StoreModule.forRoot({}),
-        StoreModule.forFeature('asm', fromReducers.getReducers()),
+        StoreModule.forFeature(ASM_FEATURE, fromReducers.getReducers()),
```

Comment:
The ASM_FEATURE was intended for lazy loading mechanism and not for the state slice.  It's probably safer not to use it to to identify state slices.